daq_lib.py

```diff
@@ -753,8 +754,22 @@ def collectData(currentRequest):
           node = getBlConfig(nodeName)      
           dimpleNode = getBlConfig("dimpleNode")      
           if (daq_utils.detector_id == "EIGER-16"):
+            with open(f"{os.environ['CONFIGDIR']}/autoproc_blacklist.json") as autoproc_blacklist_file:
```

Comment:
could this be moved somewhere so that the file does not have to be json.loaded every time the check is performed?